sklearn/linear_model/_glm/glm.py

```diff
@@ -622,6 +622,11 @@ class GammaRegressor(GeneralizedLinearRegressor):
     2.896...
     >>> clf.predict([[1, 0], [2, 8]])
     array([19.483..., 35.795...])
+
+    See Also
+    ----------
+    GeneralizedLinearRegressor : Generalized Linear Model with a Gamma
+        distribution.
```

Comment:
This base class is not meant to be used directly. Instead you could link to `PoissonRegressor`, `TweedieRegressor` and `Ridge` for models respectively with a Poisson, Tweedie and Gaussian distributed target variable.